scripts/CMakeLists.txt

```diff
@@ -452,6 +452,10 @@ ELSE()
       )
   ENDIF()
 
+  IF(WITH_ROCKSDB)
+    LIST(APPEND BIN_SCRIPTS percona_server_admin)
```

Comment:
In a previous discussion, I believe PZ preferred the the more verbose percona_server to avoid any ambiguity with other tools and also as a bit of a marketing thing to keep the full Percona brand recognizable. Not disagreeing with you, just putting the info out there.